docs/images/bluefin/developer-experience.md

```diff
@@ -20,7 +20,11 @@ Like all Universal Blue images, switching is atomic, allowing for clean switchin
 
 ### Visual Studio Code
 
-[Visual Studio Code](https://code.visualstudio.com/) is included on the image to facilitate local development.
+[Visual Studio Code](https://code.visualstudio.com/) is included on the image to facilitate local development. It comes configured for usage with [devcontainers] and Podman via a small [default configuration file](https://github.com/ublue-os/bluefin/blob/main/dx/usr/etc/skel.d/.config/Code/User/settings.json).
+
+- [Dev Containers Documentation](https://code.visualstudio.com/docs/devcontainers/containers) - you can skip most of the installation instructions and go directly to [the tutorial](https://code.visualstudio.com/docs/devcontainers/tutorial)
```

Comment:
```suggestion
- [Dev Containers Documentation](https://code.visualstudio.com/docs/devcontainers/containers) - you can skip most of the installation instructions and go directly to [the tutorial](https://code.visualstudio.com/docs/devcontainers/tutorial#_install-the-extension)
I would point them just over the "check docker version stage" right into the install extention, because we set podman by default.
```